crates/paralegal-policy/src/flows_to.rs

```diff
@@ -172,10 +172,56 @@ fn test_data_flows_to() {
             })
             .unwrap()
     };
-    let sink1 = get_sink("sink1");
-    let sink2 = get_sink("sink2");
-    assert!(ctx.data_flows_to(controller, &src, sink1));
-    assert!(!ctx.data_flows_to(controller, &src, sink2));
+    let sink1: CallSiteOrDataSink = get_sink("sink1").into();
+    let sink2: CallSiteOrDataSink = get_sink("sink2").into();
+    assert!(ctx.flows_to(
+        Some(controller.clone()),
+        &src,
+        &sink1,
+        crate::EdgeType::Data
+    ));
+    assert!(!ctx.flows_to(
+        Some(controller.clone()),
+        &src,
+        &sink2,
+        crate::EdgeType::Data
+    ));
+}
+
+#[test]
+fn test_ctrl_flows_to() {
```

Comment:
Thanks for adding some test cases, I think it would make sense to add a few assertions to it about flows that _shouldn't_ exist.

Reply:
done